elife/php7.sls

```diff
@@ -46,6 +46,7 @@ php-log:
         - name: /var/log/php_errors.log
         - user: {{ pillar.elife.webserver.username }}
         - mode: 640
+        - replace: False
```

Comment:
the intent of this state is that the log file simply exists, creating if it doesn't:
`https://docs.saltstack.com/en/latest/ref/states/all/salt.states.file.html#salt.states.file.managed`

if the `source` attribute is absent it does just that. Is there a reason for adding the `replace` attribute?